<launch>
  <!-- Other nodes or includes can go here -->
  <node pkg="ir2324_group_24" type="Node_A" name="Node_A" output="screen"/>
  <node pkg="ir2324_group_24" type="Node_B" name="Node_B" output="screen"/>

  <!-- Load Global Costmap Parameters -->
  <rosparam file="$(find ir2324_group_24)/config/global_costmap.yaml" command="load" />

  <!-- Load Local Costmap Parameters -->
  <rosparam file="$(find ir2324_group_24)/config/local_costmap.yaml" command="load" />

  <!-- Load Base Local Planner Parameters (if any) -->
  <rosparam file="$(find ir2324_group_24)/config/base_local_planner_params.yaml" command="load" />

  <!-- Start move_base Node -->
  <node name="move_base" pkg="move_base" type="move_base" respawn="false" output="screen">
    <!-- Remappings or additional parameters can be specified here -->
  </node>
</launch>
